src/lib/components/table.svelte

```diff
@@ -19,9 +20,12 @@
     let startAt = 0
     let endAt = pageSize
     let visibleResults
+    let showLoadMoreButton = false
 
     $: data?.rows, updateVisibleResults()
 
+    export let loadMore = (..._: any[]): void => {}
```

Comment:
Maybe you can move this a bit up to be next to `export let data` 🙏🏼 ? Just to keep the code clean

Reply:
sure thanks